src/blocks/Hero/Hero.js

```diff
@@ -63,6 +66,8 @@ const HeroButton = styled(Button, {
     ...theme.mixins.absolute(0),
     content: '""',
   },
+  margin: '0 20px',
```

Comment:
I would suggest looking up `theme.spacing`, link: https://mui.com/material-ui/customization/spacing/

theme.spacing has a default value of 8px (don't think we've overwritten it) and you can use it for margins and paddings to get a more theme-related layout instead of using pxls. 

You can use theme.spacing in the same way as you've done in margin and use theme.spacing('val1', 'val2')